### PR TITLE
K8SPS-364: Reconcile full cluster crush only for GR cluster type.

### PR DESCRIPTION
**Pull request by inelpandzic**
[K8SPS-364](https://jira.percona.com/browse/K8SPS-364) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
We were reconciling full cluster crash for async clusters as well.

**Solution:**
Reconcile full cluster crush only for GR cluster type.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PS version?
- [x] Does the change support oldest and newest supported Kubernetes version?

[K8SPS-364]: https://perconadev.atlassian.net/browse/K8SPS-364?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-demand-backup-haproxy.log) |
| gr-finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-finalizer.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-scaling.log) |
| gr-security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-security-context.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/122cec2a/version-service.log) |
| We run 32 out of 32|

commit: https://github.com/percona/percona-server-mysql-operator/pull/709/commits/122cec2aea68e6e65dbbcaa467b2229bac1d926a
image: `perconalab/percona-server-mysql-operator:PR-709-122cec2a`

**Review by hors — APPROVED** [review]

File: pkg/controller/ps/crash_recovery.go [hunk 1]
```diff
@@ -19,6 +19,10 @@ import (
 func (r *PerconaServerMySQLReconciler) reconcileFullClusterCrash(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL) error {
 	log := logf.FromContext(ctx).WithName("Crash recovery")
 
+	if cr.Spec.MySQL.ClusterType == apiv1alpha1.ClusterTypeAsync || cr.OrchestratorEnabled() {
```
Comment (egegunes) [inline pkg/controller/ps/crash_recovery.go:22]: why `|| cr.OrchestratorEnabled()` ?

**Review by egegunes — CHANGES_REQUESTED** [review]

File: pkg/controller/ps/crash_recovery.go (hunk 1 above)
Comment (inelpandzic) [reply pkg/controller/ps/crash_recovery.go]: No good reason: [1651559](https://github.com/percona/percona-server-mysql-operator/pull/709/commits/16515590a110059a8d2c135f8759dc0a838367cf)

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup-haproxy.log) |
| gr-finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-finalizer.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-scaling.log) |
| gr-security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-security-context.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/users.log) |
| version-service | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/version-service.log) |
| We run 32 out of 32|

commit: https://github.com/percona/percona-server-mysql-operator/pull/709/commits/16515590a110059a8d2c135f8759dc0a838367cf
image: `perconalab/percona-server-mysql-operator:PR-709-16515590`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup-haproxy.log) |
| gr-finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-finalizer.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-scaling.log) |
| gr-security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-security-context.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/users.log) |
| version-service | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/version-service.log) |
| We run 32 out of 32|

commit: https://github.com/percona/percona-server-mysql-operator/pull/709/commits/16515590a110059a8d2c135f8759dc0a838367cf
image: `perconalab/percona-server-mysql-operator:PR-709-16515590`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup.log) |
| gr-demand-backup-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-demand-backup-haproxy.log) |
| gr-finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-finalizer.log) |
| gr-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-haproxy.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-init-deploy.log) |
| gr-one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-one-pod.log) |
| gr-recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-recreate.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-scaling.log) |
| gr-security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-security-context.log) |
| gr-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-self-healing.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-tls-cert-manager.log) |
| gr-users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/gr-users.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/one-pod.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/operator-self-healing.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/recreate.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/sidecars.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/smart-update.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-709/16515590/version-service.log) |
| We run 32 out of 32|

commit: https://github.com/percona/percona-server-mysql-operator/pull/709/commits/16515590a110059a8d2c135f8759dc0a838367cf
image: `perconalab/percona-server-mysql-operator:PR-709-16515590`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**